.github/workflows/build.yaml

```diff
@@ -176,7 +181,8 @@ jobs:
   # (see https://github.com/actions/runner/issues/952 and https://github.com/orgs/community/discussions/9141)
   summary:
     name: CI Results Summary
-    needs: [testsuite, windows, macos, appimage]  # We do not require MSVC due to frequent unrelated runner failures
+    # "changes" has to be included in needs here too, because all others are skipped if it fails
+    needs: [changes, testsuite, windows, windows-msvc, macos, appimage]
```

Comment:
So, do you consider msvc stable now? At least recently it was more stable than mingw...

Reply:
Yes, we have now mitigations for instabilities in both. And also we hadn't excluded it before either. I just forgot to request removing that exception from the previous PR in time.